MetaData/python/data13TeV_LFV.py

```diff
@@ -101,6 +101,73 @@ def xsbr(*args, **kwargs):
     'calibrationTarget': 'RunIISpring15DR7',
     'x_sec': 18610.0 ,
 }
+###new stuff
+datadefs["DY1JetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-pythia8"] = {
+    'analyses': ['4L'],
+    'datasetpath': '/DYJetsToLL_M-10to50_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8/RunIISpring15DR74-Asympt25ns_MCRUN2_74_V9-v1/MINIAODSIM',
+    'pu': 'Asympt25ns',
+    'calibrationTarget': 'RunIISpring15DR7',
+    'x_sec': 1016 ,
+}
```

Comment:
For future use, this is a nice link to keep around for cross sections:
https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns

On Wed, Feb 10, 2016 at 9:10 PM, Nabarun Dev <notifications@github.com>
wrote:

> In MetaData/python/data13TeV_LFV.py
> <https://github.com/uwcms/FinalStateAnalysis/pull/494#discussion_r52517430>
> :
>
> > @@ -101,6 +101,73 @@ def xsbr(*args, **kwargs):
> >      'calibrationTarget': 'RunIISpring15DR7',
> >      'x_sec': 18610.0 ,
> >  }
> > +###new stuff
> > +datadefs["DY1JetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-pythia8"] = {
> > +    'analyses': ['4L'],
> > +    'datasetpath': '/DYJetsToLL_M-10to50_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8/RunIISpring15DR74-Asympt25ns_MCRUN2_74_V9-v1/MINIAODSIM',
> > +    'pu': 'Asympt25ns',
> > +    'calibrationTarget': 'RunIISpring15DR7',
> > +    'x_sec': 1016 ,
> > +}
>
> I see. Thank you.I will multiply each by k=1.23 and update.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uwcms/FinalStateAnalysis/pull/494/files#r52517430>.
>